kettle/make_json.py

```diff
@@ -162,41 +165,55 @@ def get_metadata():
     return build
 
 
+def get_table(days):
+    if days:
+        return ('build_emitted_%g' % days).replace('.', '_')
+    return 'build_emitted'
+
+
 def parse_args(args):
     parser = argparse.ArgumentParser()
-    parser.add_argument('--day', action='store_true',
-                        help='Grab data for builds in the last day')
-    parser.add_argument('--days', type=float,
+    parser.add_argument('--days', type=float, default=0,
                         help='Grab data for builds within N days')
     parser.add_argument('--reset-emitted', action='store_true',
                         help='Clear list of already-emitted builds.')
+    parser.add_argument('paths', nargs='*',
```

Comment:
Can you break all these after `add_argument(`? I suspect they'll all fit on one line then